### PR TITLE
[Cody GA Docs] Final PR 

### PR DESCRIPTION
**Pull request by MaedahBatool**
This is the final docs PR scheduled to be merged on Dec 14th. It has the following changes:

- Overview
   - Remove the `Beta` label from the main homepage
   - Fix label naming in `Link` cards
   - Link for use cases in the `More Resources` section

- Cody for VS Code
   - Update NLS image under `Verifying the installation` — _To Do_
   - LLM Selection limits — _To Do_
   - `Beta` label under Custom Commands section
   - `Beta` label under NLS section
   - Fix the link for Commands in `More Resources` section

- Cody for JetBrains
   - Update link in the `More Resources `for `Use Cases` 

- Cody for Neovim
   - Fix the link card at the beginning
   - Update link in the `More Resources `for `Use Cases` 

- Cody for Web
   - Update link in the `More Resources `for `Use Cases` 

- Cody App
   - Remove the link from the left nav

- Feature Parity Reference
   - Move it under Overview with other Cody Clients

- Cody Quickstart
   - Update NLS image with Beta label — _To Do_

- Cody Capabilities — Commands
   - Add Beta label for Custom Commands

- Cody Pricing (moved content from [this PR](https://github.com/sourcegraph/sourcegraph/pull/58574))
   - Add a section in the side nav called `Usage and Pricing`
   - Add usage and pricing docs 

## Preview 🤩
[Preview Link](https://docs.sourcegraph.com/@dec-14)

### CODE REVIEW

File: doc/cody/usage-and-pricing.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+To revert back to Cody Free from Pro:
+
+- Go to your Sourcegraph dashboard **Cody > Manage**
+- Next, **Manage subscription** that takes you to **Cody > Subscription**
+- Clicks **Cancel** on the Pro tier to cancel your Pro subscription
+- This automatically downgrades you to Cody Free
+
+### Upgrading from Pro to Enterprise
+
+To upgrade from Cody Pro to Cody Enterprise, you should [Contact Sales](https://sourcegraph.com/contact/request-info) and connect with one of our support assistants. They will help you set up your account and start with Cody Enterprise.
```
Comment (ryphil) [inline doc/cody/usage-and-pricing.md:48]: "connect with one of our account teams." is more accurate.

File: doc/cody/usage-and-pricing.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Enterprise
+
+Cody Enterprise is designed for enterprises prioritizing security and administrative controls, starting at a consumption-based pricing capped at $19 per user per month. You get additional capabilities like BYOLLM (Bring Your Own LLM), supporting Single-Tenant and Self Hosted setups for flexible coding environments.
```
Comment (ryphil) [inline doc/cody/usage-and-pricing.md:52]: What does this mean "consumption based pricing capped at $19/u/m"? 

I would say "we offer either seat-based or token based pricing models, depending on what makes most sense for your organization".

File: doc/cody/usage-and-pricing.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Security features include SAML/SSO for enhanced authentication, audit logs for transparency, and guardrails to enforce coding standards. Cody Enterprise supports advanced Code Graph context and multi-code host context for a deeper understanding of codebases, especially in complex projects. With 24/5 enhanced support, Cody Enterprise ensures timely assistance.
```
Comment (ryphil) [inline doc/cody/usage-and-pricing.md:54]: Audit logs for Cody are not available until February, not sure if this is misleading including it here.

Comment (MaedahBatool) [reply doc/cody/usage-and-pricing.md:54]: I'll remove it for now and then include in Feb release. 🙌

File: doc/cody/capabilities/commands.md [hunk 4]
```diff
@@ -1,42 +1,47 @@
...
-The support for one command may vary by IDE extension. Read the [feature parity reference to learn what commands are available in your IDE](./../feature-reference.md#commands).
+Support for commands may vary by IDE extension. Read the [feature parity reference to learn what commands are available in your IDE](./../feature-reference.md#commands).
 
-> NOTE: Cody's responses to commands might be better than responses to chats since they've been pre-packaged and prompt-engineered.
+> NOTE: Cody’s responses to commands are often better than responses to chats since they’ve been pre-packaged and prompt-engineered.
 
 ## Running commands
 
-The process of running a command varies from one IDE to another. For example, in VS Code, there are three ways to run a command:
+The process of running a command varies from one IDE to another. For example, in VS Code, there several ways to run a command:
```
Comment (taylorsperry) [inline doc/cody/capabilities/commands.md:21]: there *are* several ways . . .

File: doc/cody/capabilities/commands.md [hunk 5]
```diff
@@ -1,42 +1,47 @@
...
+1. Highlight your code and select the command from the sidebar
 
-Commands are also available in the JetBrains extension. To access it, navigate to the `Commands` panel (next to the `Chat` panel), and you can find each available command as a button within the UI.
+![running-commands](https://storage.googleapis.com/sourcegraph-assets/Docs/using-commands.png)
+
+2. Open up the command palette with `Option+C` `Alt+C`
+3. Right-click on any code element and select **Cody > Choose a command** from the list
+4. Type `/` in the chat bar. Cody will then suggest a list of available commands
+
+To access it, navigate to the `Commands` panel (next to the `Chat` panel), and you can find each available command as a button within the UI.
```
Comment (taylorsperry) [inline doc/cody/capabilities/commands.md:31]: Is this last sentence (To access it, navigate to the `Commands` panel (next to the `Chat` panel), and you can find each available command as a button within the UI) redundant to the list above?

File: doc/cody/overview/install-vscode.md [hunk 6]
```diff
@@ -158,10 +158,9 @@ The following tables shows what happens when Enhanced Context Selection is enabl
 | Enhanced Context Enabled  | ✅                           | ✅                          | ✅                          | ✅                          |
 | Enhanced Context Disabled | ❌                           | ❌                          | ❌                          | ❌                          |
 
-
 ### LLM Selection
 
-Cody Community users can choose the LLM they'd like Cody to use for chats right within the chat panel. The default LLM is set to Claude 2.0 by Anthropic, but the drop-down allows you to experiment with different LLMs and choose the one that's best for you. For Enterprise users, the LLM is determined by the administrator and cannot be changed within the editor.
+Cody Free users can choose the LLM they'd like Cody to use for chats right within the chat panel. The default LLM is set to Claude 2.0 by Anthropic, but the drop-down allows you to experiment with different LLMs and choose the one that's best for you. Users on Cody Pro also get to select Mixtral with all other LLMs suppported for Free users. For Enterprise users, the LLM is determined by the administrator and cannot be changed within the editor.
```
Comment (taylorsperry) [inline doc/cody/overview/install-vscode.md:163]: Cody *Pro* users can choose the LLM they'd like Cody to use for chats.

(Cut "Users on Cody Pro also get to select Mixtral with all other LLMs supported for Free users.)

File: doc/cody/quickstart.md [hunk 7]
```diff
@@ -126,7 +130,8 @@ Leveraging the failed test output, Cody is able to identify the potential bug an
 
 ## 3. Ask Cody to add code documentation
 
-Cody can also create well-crafted code documentation and comments, significantly improving the readability experience. You can add it by selecting a code snippet and clicking **Document** from the **Commands** panel. Alternatively, select the code, and inside the Cody chat window, type `/doc`. Cody will add a comment to the top of the code snippet.
+Cody can also create well-crafted code documentation and comments, significantly improving the readability experience.
+You can add it by selecting a code snippet and running the **Document** command. Cody will add a comment to the top of the code snippet. Alternatively, select the code, and inside the Cody chat window, type `/doc`. Cody will add a comment to the top of the code snippet.
```
Comment (taylorsperry) [inline doc/cody/quickstart.md:134]: Cut: "Alternatively, select the code, and inside the Cody chat window, type `/doc`. Cody will add a comment to the top of the code snippet."

(/doc can't be triggered from chats anymore)

**Review by taylorsperry — APPROVED** [review]
Last couple comments! Thank you, @MaedahBatool !

File: doc/cody/usage-and-pricing.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+Cody Pro, designed for individuals or small teams at **$9 per user per month**, offers an enhanced coding experience beyond the free plan. It provides unlimited autocompletion suggestions, allowing users to streamline their coding process without restrictions. Chat and commands executions are also unlimited, allowing users to create custom workflows.
+
+The plan includes the ability to create an unlimited number of Custom Commands for personalized workflows. In addition to using the local context of your code to improve responses, you can embed up to **1 GB** of your private code for even better Cody responses that reflect a deep understanding of your code in VS Code, with JetBrains IDE support coming soon.
+
+Support for Cody Pro is available through Discord, ensuring prompt assistance and guidance. However, it's essential to note that the Cody Pro plan is limited to **50 developers** per month. Finally, you'll have default support with Anthropic and StarCoder as the officially supported LLMs. Moreover, Pro accounts using VS Code IDE will get an LLM selection for chat only. These LLMs are Claude Instant 1.2, Claude 2, ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, and Mixtral.
```
Comment (kalanchan) [inline doc/cody/usage-and-pricing.md:33]: > 50 developers per month

Is this accurate, I don't think gating has been in place for teams. Could we also get @chenkc805 to review this page? (disregard my comment if he already did so!)

**Review by kalanchan — COMMENTED** [review]
Kevin should also review the pricing and usage page before the merge

File: doc/cody/usage-and-pricing.md (hunk 8 above)
Comment (chenkc805) [reply doc/cody/usage-and-pricing.md:33]: We can remove "However, it's essential to note that the Cody Pro plan is limited to **50 developers** per month."

Teams functionality isn't available until Feb GA

**Pull request merged**